src/vue/modules/create-asset-sale/index.vue

```diff
@@ -0,0 +1,764 @@
+        startTime: DateUtil.toTimestamp(this.form.saleInformation.startTime),
+        endTime: DateUtil.toTimestamp(this.form.saleInformation.endTime),
+        softCap: this.form.saleInformation.softCap,
+        hardCap: this.form.saleInformation.hardCap,
+        creatorDetails: {
+          name: this.form.information.name,
+          short_description: this.form.shortBlurb.shortDescription,
+          description: saleDescriptionBlobId,
+          logo: this.form.shortBlurb.saleLogo.getDetailsForSave(),
+          youtube_video_id: this.form.shortBlurb.youtubeId,
+        },
+        requiredBaseAssetForHardCap: this.form.saleInformation.hardCap,
+        quoteAssets: this.form.saleInformation.quoteAssets
+          .map((item) => ({
+            asset: item,
+            price: '1',
+          })),
+        saleEnumType: SALE_TYPES.fixedPrice,
+        saleType: DEFAULT_SALE_TYPE,
+      }
+      return base.SaleRequestBuilder.createSaleCreationRequest(operation)
+    },
+    buildAssetRequestOperation () {
+      const requestId = this.request
+        ? this.request.id
+        : ASSET_CREATION_REQUEST_ID
+      const terms = this.form.information.terms
+
+      const operation = {
+        requestID: requestId,
+        code: this.form.information.code,
+        assetType: this.form.information.assetType.value,
+        preissuedAssetSigner: this.accountId,
+        trailingDigitsCount: this.decimalPints,
+        initialPreissuedAmount: this.form.saleInformation.hardCap,
+        maxIssuanceAmount: this.form.saleInformation.hardCap,
+        policies: this.form.information.policies.reduce((a, b) => (a | b), 0),
+        creatorDetails: {
+          name: this.form.information.name,
+          logo: EMPTY_DOCUMENT,
+          terms: terms ? terms.getDetailsForSave() : EMPTY_DOCUMENT,
+          annualReturn: this.form.saleInformation.annualReturn,
+          subtype: this.form.information.formType.value,
+        },
+      }
+      if (this.form.information.formType.value === ASSET_SUBTYPE.bond) {
+        operation.creatorDetails.maturityDate = DateUtil
+          .toMs(this.form.saleInformation.maturityDate)
+        operation.creatorDetails.logoUrl = ASSET_SUBTYPE_IMG_URL.bondLogo
+      } else {
+        operation.creatorDetails.logoUrl = ASSET_SUBTYPE_IMG_URL.shareLogo
+      }
+      return base.ManageAssetBuilder.assetCreationRequest(operation)
+    },
+    async uploadDocuments () {
+      const documents = [
+        this.form.shortBlurb.saleLogo,
+        this.form.information.terms,
+      ]
+      for (let document of documents) {
+        if (document && !document.key) {
+          const documentKey = await DocumentUploader.uploadDocument(
+            document.getDetailsForUpload()
+          )
+          document.setKey(documentKey)
+        }
+      }
+    },
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '@/vue/forms/_app-form';
+
+  .create-asset-sale-form__btn {
+    @include button-raised();
```

Comment:
Stop to use this mixin in the pair with some class. Use app__button-raised class

Reply:
ok